modules/mono/glue/GodotSharp/GodotSharp/Core/MarshalUtils.cs

```diff
@@ -5,9 +5,9 @@
 namespace Godot
 {
     using Array = Godot.Collections.Array;
-    using Dictionary = Godot.Collections.Dictionary;
+    using Dictionary = Collections.Dictionary;
```

Comment:
This is worse to me. I know that `Collections.Dictionary` does *technically* unambiguously refer to `Godot.Collections.Dictionary` because there is no `System.Collections.Dictionary` (only `System.Collections.Generic.Dictionary`), but only after putting in mental effort.

Reply:
Either is find with me. I'm surprised it only change one of those though 🤔